0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -605,25 +603,17 @@ The ledger plugin's ILP address prefix. This is used to determine whether a give
 
 The prefix may be configured, automatically detected, or hard-coded, depending on the ledger. For example, a Bitcoin ledger plugin may have the address hard-coded, while a [`five-bells-ledger`](https://github.com/interledger/five-bells-ledger) would use an API call to get the prefix.
 
-#### precision
-<code>**precision**:Number</code>
-
-The total number of digits (base 10) of precision allowed by the ledger.
-
-#### scale
-<code>**scale**:Number</code>
-
-The number of digits allowed after the decimal point.
-
 #### currencyCode
 <code>**currencyCode**:String</code>
 
-The ISO 4217 currency code (if any) used by the ledger.
+The [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) currency code (if any) used by the ledger. A custom all-caps three-letter code, not used by ISO 4217, otherwise. Ledger administrators who choose a custom currency code MAY request their chosen currency code be listed by software modules that map currency codes to currency symbols, for instance on node package manager (npm) in the case of JavaScript. The currencyCode is only meaningful in combination with the currencyScale (below).
```

Comment:
Might be worth explaining why the currencyCode is only meaningful in combination with the currencyScale